### DISCUSSION TITLE
How many FRB repeaters do we know of?

### DISCUSSION THREAD

**Discussion by 0xCoto** [chime-frb-open-data/community#54] [Q&A]
The [CHIME/FRB Catalog 1 page](https://www.chime-frb.ca/catalog) lists 18 repeaters. However, the [Repeating FRBs page](https://www.chime-frb.ca/repeaters) lists 21 sources. How up-to-date is this, and does it solely include repeaters detected by CHIME, or other telescopes too? If so, does anyone happen to know of a published catalog with more than 21 repeating sources?

Also, regarding these two repeating sources, do we have the full FRB designations/names of each repeated burst?
- https://www.chime-frb.ca/repeaters/190907.J08+46
- https://www.chime-frb.ca/repeaters/FRB20171019A

Cheers,

Apostolos

**Comment by emilypetroff** [comment]
HI Apostolos,
Looking into it a bit more on our side I see the following differences between the "Repeating FRBs page" and the "CHIME/FRB Catalog 1 page.
On the "Repeating FRBs page":
- 190907.J08+46 : First burst was detected in 2019-09, outside the date range of the CHIME/FRB Catalog 1 (also why it does not have a published TNS name).
- FRB20171019A : This FRB was originally discovered by ASKAP (https://arxiv.org/pdf/1908.10026.pdf) and the first burst from it detected by CHIME/FRB was in 2019-08, again outside the range of Catalog 1.
- FRB20200120E : Was a standalone paper outside the date range of Catalog 1.
- FRB20201124A : Was a standalone paper outside the date range of Catalog 1.

On the CHIME/FRB Catalog 1 page:
- FRB20181119D : The one and only pulse from repeating FRB20121102A (R1) discovered by CHIME/FRB. By definition this is not a repeating FRB at CHIME and thus is not on the repeating FRBs page.

As for the TNS names, we have not issued public TNS names for all the pulses from published repeaters yet. This will be done at a later date. 

Please let me know if you have other questions!
Best,
Emily
